packages/atomic/cypress/e2e/pager.cypress.ts

```diff
@@ -1,3 +1,4 @@
+import {List} from 'lodash';
```

Comment:
There's a linter error here which says that this import is never used.

You can apply this suggestion here to remove it:
```suggestion
```

Reply:
Thanks for catching this - should have been accidentally created when I was trying to refactor types